src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -24,7 +24,7 @@ const { JSON_MIME_TYPE } = require('../../util/constant');
 const updateMappingJson = (mapping) => {
   const newMapping = [];
   mapping.forEach((element) => {
-    if (get(element, 'metadata.type') && element.metadata.type === 'hashToSha256') {
+    if (get(element, 'metadata.type') && element.metadata.type.includes('hashToSha256')) {
```

Comment:
Can you explain this change?
Why are we treating `metadata.type` as array?

Reply:
In `trackConfig.json` previously we had `metadata.type = 'hashToSha256'` but now it is updated to `metadata.type = ['trim', 'hashToSha256']`.
At `constructPayload` we support both implementations as a string or as a list of string

Reply:
ok